### PR TITLE
added missed info about rt and test artifacts

### PR DESCRIPTION
**Pull request by ratoshniuk**

### CODE REVIEW

File: doc/microsite/src/main/tut/sbt/00_sbt.md [hunk 1]
```diff
@@ -353,6 +353,47 @@ You may activate this plugin with the following statement in root project:
 
 ### Transitive Artifact Downloader
 
+Working with dependecies cross projects
```
Comment (neko-kai) [inline doc/microsite/src/main/tut/sbt/00_sbt.md]: typo

File: doc/microsite/src/main/tut/sbt/00_sbt.md [hunk 2]
```diff
@@ -353,6 +353,47 @@ You may activate this plugin with the following statement in root project:
...
+---------------------------------------
+
+Descriptor allows you to search for dependencies and testscopes of your project
+
+Imagine, you have a sdk project and you want to inclide one of it's library artifacts (i.e. target_libraty).
```
Comment (neko-kai) [inline doc/microsite/src/main/tut/sbt/00_sbt.md]: This should be a part of "Create a BOM for your projects" doc
https://github.com/pshirshov/izumi-r2/pull/425/files#diff-389e35bb33460e4d05c75d0a783571f9R248

File: doc/microsite/src/main/tut/sbt/00_sbt.md [hunk 3]
```diff
@@ -353,6 +353,47 @@ You may activate this plugin with the following statement in root project:
...
+
+### R
+
+Allows you to include *Runtime artifact* of target project
+
+```
+libraryDependencies += YourSdk.R.target_library
+```
+
+### T
```
Comment (neko-kai) [inline doc/microsite/src/main/tut/sbt/00_sbt.md]: Merge all examples into one section, add the code they'll expand to.

**Review by neko-kai — CHANGES_REQUESTED** [review]

File: doc/microsite/src/main/tut/sbt/00_sbt.md [hunk 4]
```diff
@@ -263,6 +263,18 @@ lazy val myProjectBom = inRoot.as.module
 
 ```
 
+Imagine, you have a sdk project and you want to inclide one of it's library artifacts (i.e. target_libraty).
```
Comment (neko-kai) [inline doc/microsite/src/main/tut/sbt/00_sbt.md]: Make it flow with previous content in the section, drop "imagine" – describe and include code samples for generated output of BOM plugin.

File: doc/microsite/src/main/tut/sbt/00_sbt.md [hunk 5]
```diff
@@ -263,6 +263,17 @@ lazy val myProjectBom = inRoot.as.module
 
 ```
 
+If you want to include BOMs of you project, you can use the following syntax:
+
+```scala
+libraryDependencies ++= Seq(
+  YourSdk.R.my_project_bom,     // Runtime artifact
```
Comment (neko-kai) [inline doc/microsite/src/main/tut/sbt/00_sbt.md]: Make it consistent with the example given in lines above, e.g.:

```scala
MyProjectBOM.R.my_lib,     // Runtime dependency, same as `groupId %% "my-lib" % version`
MyProjectBOM.T.my_lib,     // Runtime dependency in test scope, same as `groupId %% "my-lib" % version % Test`
```

add `import com.mycompany.myproject.MyProjectBOM`

And add an example `my_lib` project that `MyProjectBOM` exports

```scala
lazy val myLib = ...
```

**Comment by ratoshniuk** [comment]
@kaishh check this out

**Comment by neko-kai** [comment]
> @kaishh check this out

LGTM

**Comment by neko-kai** [comment]
> @kaishh check this out

LGTM

**Pull request merged**